endoreg_db/services/video_import.py

```
@@ -372,243 +870,95 @@ def _ensure_frame_cleaning_available(self):
+        processor = getattr(video_meta, "processor", None) if video_meta else None
         device_name = processor.name if processor else self.processing_context['processor_name']
         
         tmp_dir = RAW_FRAME_DIR
-        output_path = Path(self.STORAGE_DIR) / f"{hash}.mp4"
+        
+        # Create temporary output path for cleaned video
+        video_filename = self.processing_context.get('video_filename', Path(raw_video_path).name)
+        cleaned_filename = f"cleaned_{video_filename}"
+        cleaned_video_path = Path(raw_video_path).parent / cleaned_filename
+        
+        processor_roi, endoscope_roi = self._get_processor_roi_info(video)
```

Pylint (4.0.0)

[error] Too many positional arguments for method call

(E1121)